api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -149,6 +149,9 @@ public void setUserProvider(Provider<User> userProvider) {
 
   @Override
   public ResponseEntity<EmptyResponse> deleteCluster(String projectName, String clusterName) {
+    User user = this.userProvider.get();
+    user.setClusterCreateRetries(0);
+    this.userDao.save(user);
```

Comment:
Nit: it's probably better to add a method to UserService to do this, and use updateWithRetries, e.g.

https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/db/dao/UserService.java#L145

That way if there's another thread saving the user at the same time you don't get an exception here.